<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>692</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>514</width>
    <height>643</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>I-V measurement</string>
  </property>
  <property name="toolTipDuration">
   <number>5</number>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="checkBox_reversesweep">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string extracomment="Test"/>
    </property>
    <property name="toolTipDuration">
     <number>5</number>
    </property>
    <property name="statusTip">
     <string extracomment="test"/>
    </property>
    <property name="text">
     <string>Reverse sweep</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Start voltage (V)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Stop voltage (V)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Voltage step (V)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Delay (s)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Compliance (A)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>71</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_Vstart">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Vstop">
       <property name="text">
        <string>-100</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Vstep">
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_delay">
       <property name="text">
        <string>0.01</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_compl">
       <property name="text">
        <string>0.002</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="asdfasdf"/>
    </property>
    <property name="text">
     <string>Start measurement</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>491</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>File name</string>
    </property>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Total current</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Breakdown</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Sensor name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_sensorname">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Output file name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_outfilename">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>339</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_date">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>339</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Output path</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_outpath">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>339</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_outpath">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>120</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_npad">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>90</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Npad</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="asdfasdf"/>
    </property>
    <property name="text">
     <string>Stop measurement</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plot">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="asdfasdf"/>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>471</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>557</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actiontest"/>
    <addaction name="separator"/>
    <addaction name="actiontest23"/>
   </widget>
   <addaction name="menu_file"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiontest">
   <property name="text">
    <string>&amp;New measurement</string>
   </property>
   <property name="iconText">
    <string>&amp;New measurement</string>
   </property>
  </action>
  <action name="actiontest23">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
